### PR TITLE
NXP-23871: Pick up variables from the route/workflow instance

### PR DESCRIPTION
**Pull request by covolution**
PR created from https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-gethin/53/

### CODE REVIEW

File: nuxeo-routing-web/src/main/java/org/nuxeo/ecm/platform/routing/web/RoutingTaskActionsBean.java [hunk 1]
```diff
@@ -395,7 +395,7 @@ protected String getTaskActionId(Task task, String buttonId) {
     public Map<String, Action> getTaskActionsMap(Task task) {
         Map<String, Action> actions = new LinkedHashMap<String, Action>();
         // bulk processing, don't fetch formVariables to avoid overriding them
```
Comment (efge) [inline nuxeo-routing-web/src/main/java/org/nuxeo/ecm/platform/routing/web/RoutingTaskActionsBean.java]: Is this comment still meaningful?

**Comment by efge** [comment]
Why is this already pushed? https://github.com/nuxeo/nuxeo-platform-document-routing/commit/c7389afdc77284ef7cde5780606b24956bb7e5f5

**Comment by efge** [comment]
👍

**Comment by covolution** [comment]
Already merged.

**Pull request closed without merging**